src/components/table/types.ts

```diff
@@ -0,0 +1,48 @@
+import { ReactNode } from 'react';
+
+export interface Column {
+  key: string;
+  header: string;
+}
+export interface PrimaryColumn extends Column {
+  primary: boolean;
+}
+export interface FixedColumn extends Column {
+  width: number;
```

Comment:
Is it possible to make it optional and accept not only pixels?
In case it is set as an absolute value in pixels, it would be impossible to split the table into two/three columns with 50%/33% width without knowing its maximum size,

Reply:
Fixed Columns width isn't optional, cause they don't have min-width
Will fix type